apps/fabric-website/src/pages/Styles/LocalizationPage/docs/web/LocalizationRTL.md

```diff
@@ -1,15 +1,14 @@
-Fabric comes with an alternate CSS file for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. This reverses the order of columns in the responsive grid, making it easy to create an RTL layout without writing additional templates. Many icons are also reversed, particularly those used for navigation such as arrows.
+Fabric comes with built-in support for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. Fabric achieves this by reversing the order of columns in the responsive grid, making it easy to create an RTL layout without writing additional templates. In addition to this, Fabric Core specifically also reverses many icons, particularly those used for navigation such as arrows.
```

Comment:
Probably good to be clearer about what's part of Fabric React vs. Core. Something like this:

```suggestion
Fabric comes with built-in support for pages written in right-to-left (RTL) languages, such as Arabic and Hebrew. 

Each component in Fabric React will automatically reverse the layout of elements within the component.

For pages using Fabric Core's responsive grid, the order of columns is automatically reversed, making it easy to create an RTL layout without writing additional templates. Fabric Core also reverses many icons, particularly those used for navigation such as arrows. (Icons in Fabric React are not automatically reversed.)
```